package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tgui-core",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
Updating the version is now done via Release, per https://github.com/tgstation/tgui-core#releasing-a-new-version